mithril-aggregator/src/http_server/routes/artifact_routes/snapshot.rs

```diff
@@ -188,7 +188,7 @@ mod handlers {
         config: Configuration,
         signed_entity_service: Arc<dyn SignedEntityService>,
     ) -> Result<impl warp::Reply, Infallible> {
-        debug!(logger, "⇄ HTTP SERVER: snapshot_download/{}", digest);
+        debug!(logger, "GET /snapshot_download/snapshot/{digest}");
```

Comment:
```suggestion
        debug!(logger, "GET /artifact/snapshot/{digest}/download");
```

Reply:
Log removed in favor of a general route entry log system !